packages/reports/app/styles/navi-reports/components/filter-builder.scss

```diff
@@ -19,6 +19,19 @@ $filter-builder-padding: 4px;
       display: flex;
       line-height: 1;
       min-height: $fields-sizes-md-height;
+
+      .ember-power-select-trigger {
```

Comment:
Can we add this to the `dropdown-parameter-picker.scss` file with the component class so we don't need the `!important` e.g.: 
```scss
.dropdown-parameter-picker {
  .ember-power-select-trigger {
   ...
  }
}